docs/src/man/getting_started.md

```diff
@@ -124,37 +124,63 @@ It is also possible to fill a `DataFrame` row by row. Let us construct an empty
 julia> df = DataFrame(A = Int[], B = String[])
 0×2 DataFrame
 ```
+Additionally, a `DataFrame` object can be constructed from an array of named tuples, where each tuple of the array corresponds to a row of the new data frame. The order and types of the elements in each of the named tuples specify the order of the columns and their types:
+
+```jldoctest dataframe
+julia> r1 = (A = 1, B = "M")
+(A = 1, B = "M")
+
+julia> r2 = (A = 2, B = "N")
+(A = 2, B = "N")
+
+julia> df = DataFrame([r1, r2])
```

Comment:
```suggestion
julia> df = DataFrame([(A = 1, B = "M"),
                       (A = 2, B = "N")])
```

Reply:
I think we do not need to show creation of `r1` and `r2` explicitly. Can you also please remove the respective lines above?